src/components/Navigation.jsx

```diff
@@ -27,37 +65,62 @@ class Navigation extends Component {
-              <Nav.Link href="https://imaginethisucl.github.io/getting%20started/how%20to%20use.html">
-                Get Started
-              </Nav.Link>
-              <Nav.Link href="https://imaginethisucl.github.io/guidelines/design%20introduction.html">
-                Guidelines
-              </Nav.Link>
+            <Nav className="">
+              <Nav.Link
+                href="https://imaginethisucl.github.io/getting%20started/how%20to%20use.html">Get
+                Started</Nav.Link>
+              <Nav.Link
+                href="https://imaginethisucl.github.io/guidelines/design%20introduction.html">Guidelines</Nav.Link>
+
             </Nav>
-            <Nav>
+            <Nav className="mr-auto">
               <NavDropdown
                 alignRight
                 title="GitHub"
                 id="collapsible-nav-dropdown"
               >
-                <NavDropdown.Item href="https://github.com/ImagineThisUCL/ImagineThisWeb">
-                  Web App
-                </NavDropdown.Item>
-                <NavDropdown.Item href="https://github.com/ImagineThisUCL/ImagineThisServer">
-                  Server
-                </NavDropdown.Item>
-                <NavDropdown.Item href="https://github.com/ImagineThisUCL/ImagineThis-Mobile">
-                  Mobile Components
-                </NavDropdown.Item>
+                <NavDropdown.Item
+                  href="https://github.com/ImagineThisUCL/ImagineThisWeb">Web
+                  App</NavDropdown.Item>
+                <NavDropdown.Item
+                  href="https://github.com/ImagineThisUCL/ImagineThisServer">Server</NavDropdown.Item>
+                <NavDropdown.Item
+                  href="https://github.com/ImagineThisUCL/ImagineThis-Mobile">Mobile
+                  Components</NavDropdown.Item>
               </NavDropdown>
             </Nav>
+
+            <Nav>
+              <Form onSubmit={this.handleSubmit} className="input-group">
+                <InputGroup className="input-group-prepend">
+                  <FormControl
+                    className="form-control navbar-input"
+                    aria-describedby="basic-addon1"
+                    placeholder="Find Project With ID"
+                    value={this.state.value}
+                    onChange={this.handleChange}
+                  />
+                  <InputGroup.Append>
+                    <Button variant="btn btn-light search-button" type="submit">
+                      <svg xmlns="http://www.w3.org/2000/svg"
+                           width="16" height="16"
+                           fill="currentColor"
+                           className="bi bi-search"
+                           viewBox="0 0 16 16">
+                        <path
+                          d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
```

Comment:
Might be better to move the SVG into the assets folder. Would help with reusability and in case we'd watch to change it.